dev/bots/test.dart

```diff
@@ -1547,6 +1548,43 @@ Future<void> _runFlutterPackagesTests() async {
   });
 }
 
+// Runs customer_testing.
+Future<void> _runCustomerTesting() async {
+  printProgress('${green}Running customer testing$reset');
+  await runCommand(
+    'git',
+    <String>[
+      'fetch',
+      'origin',
+      'master',
+    ],
+    workingDirectory: flutterRoot,
+  );
+  await runCommand(
+    'git',
+    <String>[
+      'checkout',
+      'master',
+    ],
+    workingDirectory: flutterRoot,
+  );
+  final Map<String, String> env = Platform.environment;
+  final String revision = env['REVISION'] ?? 'HEAD';
+  await runCommand(
+    'git',
+    <String>[
+      'checkout',
+      revision,
+    ],
+    workingDirectory: flutterRoot,
+  );
```

Comment:
looks like `git checkout HEAD` is a no-op (HEAD is what we have currently checked out), thus let's only run the command if `revision != null`:

```suggestion
  final String? revision = env['REVISION'];
  if (revision != null) {
    await runCommand(
      'git',
      const <String>[
        'checkout',
        revision,
      ],
      workingDirectory: flutterRoot,
    );
  }
```

Reply:
in what cases would `env['REVISION']` not be set? just when run locally?

Reply:
> looks like `git checkout HEAD` is a no-op (HEAD is what we have currently checked out), thus let's only run the command if `revision != null`:

Applied the suggestion.